### PR TITLE
Create new table for location specific latest analytics.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**
This will eventually replace the previous table which will be deleted on switchover.

### CODE REVIEW

File: cloudformation/dynamoDb.yaml [hunk 1]
```diff
@@ -240,6 +240,32 @@ Resources:
...
+    Properties:
+      AttributeDefinitions:
+      - AttributeName: territory
+        AttributeType: S
+      - AttributeName: campaignId
+        AttributeType: S
+      KeySchema:
+      - AttributeName: territory
+        KeyType: HASH
+      - AttributeName: campaignId
```
Comment (janua) [inline cloudformation/dynamoDb.yaml:162]: Which way around is better, is this going to be displayed inside a particular campaign? If so maybe the opposite might be better?

**Review by JonNorman — APPROVED** [review]

File: cloudformation/dynamoDb.yaml (hunk 1 above)
Comment (LATaylor-guardian) [reply cloudformation/dynamoDb.yaml:162]: @janua I thought about this for a bit. I think this way as Campaign Central will be contextual to the territory which the user will select at the top level of Campaign Central. So this allows you to query for all campaigns in a given territory.

Comment (janua) [reply cloudformation/dynamoDb.yaml:162]: Ah I see... Then you can batch get the campaigns!

Comment (LATaylor-guardian) [reply cloudformation/dynamoDb.yaml:162]: Yeah :)

**Comment by janua** [comment]
:+1:

**Pull request merged**